extractor/src/extractors/ExtractorRegistry.ts

```diff
@@ -11,6 +11,7 @@ import IvyXmlExtractor from "./IvyXmlExtractor";
 import PackageJsonExtractor from "./PackageJsonExtractor";
 import PomXmlExtractor from "./PomXmlExtractor";
 import VendorConfExtractor from "./VendorConfExtractor";
+import JsonnetJsonExtractor from "./JsonnetJsonExtractor";
```

Comment:
Let's call this `JsonnetfileJsonExtractor` for consistency. I know it's a bit wordy